tools/ipfs-upload/bin/upload-remix-plugin

```diff
@@ -4,9 +4,20 @@ const ipfsClient = require('ipfs-http-client')
 const signale = require('signale')
 const generateProfile = require('../generateProfile')
 
+var program = require('commander')
+var fs = require('fs')
+var path = require('path')
+
+program
+.description('Upload resources to IPFS and generate a profile.')
+.option('--profile-path  <path>', 'Path where the profile.json should be generated. Default to current path')
+.parse(process.argv)
+
 const host = 'ipfs.komputing.org' // ethdev berlin ipfs node
 const ipfs = ipfsClient({ host, port: 443, protocol: 'https' })
 
+const targetDir = program.profilePath || __dirname
```

Comment:
__dirname here is the folder where the `upload-remix-plugin` is ? I don't think this is the expected behaviour. I would recommand adding a prompt to ask the path if not provided.

Reply:
added this for tracking it https://github.com/ethereum/remix-plugins-directory/issues/16